pkg/hub/submarineragent/controller.go

```diff
@@ -231,22 +252,27 @@ func (c *submarinerAgentController) syncManagedCluster(
 	ctx context.Context,
 	managedCluster *clusterv1.ManagedCluster,
 	config *configv1alpha1.SubmarinerConfig) error {
-	// the cluster does not have the submariner label, try to clean up the submariner agent
-	if _, existed := managedCluster.Labels[submarinerLabel]; !existed {
-		return c.cleanUpSubmarinerAgent(ctx, managedCluster)
-	}
-
-	// the cluster does not have the clusterset label, try to clean up the submariner agent
 	clusterSetName, existed := managedCluster.Labels[clusterSetLabel]
```

Comment:
do we generete clusterset claim in managedcluster? we should rely on that instead of calling clusterset on hub.

Reply:
This is hub part, so we don't need to generate claim